templates/macros/assets.html

```diff
@@ -1,56 +1,94 @@
+    <path fill-rule="evenodd"
+        d="M8.75.75a.75.75 0 00-1.5 0V2h-.984c-.305 0-.604.08-.869.23l-1.288.737A.25.25 0 013.984 3H1.75a.75.75 0 000 1.5h.428L.066 9.192a.75.75 0 00.154.838l.53-.53-.53.53v.001l.002.002.002.002.006.006.016.015.045.04a3.514 3.514 0 00.686.45A4.492 4.492 0 003 11c.88 0 1.556-.22 2.023-.454a3.515 3.515 0 00.686-.45l.045-.04.016-.015.006-.006.002-.002.001-.002L5.25 9.5l.53.53a.75.75 0 00.154-.838L3.822 4.5h.162c.305 0 .604-.08.869-.23l1.289-.737a.25.25 0 01.124-.033h.984V13h-2.5a.75.75 0 000 1.5h6.5a.75.75 0 000-1.5h-2.5V3.5h.984a.25.25 0 01.124.033l1.29.736c.264.152.563.231.868.231h.162l-2.112 4.692a.75.75 0 00.154.838l.53-.53-.53.53v.001l.002.002.002.002.006.006.016.015.045.04a3.517 3.517 0 00.686.45A4.492 4.492 0 0013 11c.88 0 1.556-.22 2.023-.454a3.512 3.512 0 00.686-.45l.045-.04.01-.01.006-.005.006-.006.002-.002.001-.002-.529-.531.53.53a.75.75 0 00.154-.838L13.823 4.5h.427a.75.75 0 000-1.5h-2.234a.25.25 0 01-.124-.033l-1.29-.736A1.75 1.75 0 009.735 2H8.75V.75zM1.695 9.227c.285.135.718.273 1.305.273s1.02-.138 1.305-.273L3 6.327l-1.305 2.9zm10 0c.285.135.718.273 1.305.273s1.02-.138 1.305-.273L13 6.327l-1.305 2.9z">
+    </path>
+</svg>
+{% endmacro license_logo_svg %}
+
+
 {% macro card(post) %}
-<div class="card" id="{{ post.title | slugify }}">
-    <a href="{{ post.extra.link }}">
-        {% if post.extra.image %}
-        <div class="card-image card-image-default">
+<div id="{{ post.title | slugify }}">
+    <a class="asset-card" href="{{ post.extra.link }}">
+        <div class="asset-card__banner">
+            <div class="asset-card__title">{{post.title}}</div>
+            {% if post.extra.image %}
             <img src="{{ image_macros::resize_image(path=post.extra.image, width=340, height=340) }}" />
+            {% endif %}
         </div>
-        {% endif %}
-        <div class="card-text">
-            <div class="card-title">{{ post.title }}</div>
-            <div class="card-description">
-                {{ post.description | striptags | truncate(length=100) | safe }}
+        <div class="asset-card__description">
+            {{ post.description | striptags | truncate(length=120) | safe }}
+        </div>
+        <div class="asset-card__tags">
+            {% if post.extra.bevy_versions %}
+            <div class="asset-card__tag-list asset-card__bevy-versions">
+                <img src="/assets/bevy_icon_dark.svg" alt="Small bevy icon">
+                {% for version in post.extra.bevy_versions %}
+                <span class="asset-card__tag">{{version}}</span>
+                {% endfor %}
+            </div>
+            {% endif %}
+            {% if post.extra.licenses %}
+            <div class="asset-card__tag-list asset-card__licenses">
+                {{ assets_macros::license_logo_svg() }}
+                {% for license in post.extra.licenses %}
+                <span class="asset-card__tag">{{license}}</span>
```

Comment:
Similar here, except that `alt` attributes don't work on `svg`s.
From what I've read, the best way to do this is to add a `title` element inside the `svg`: [Blog post](https://css-tricks.com/accessible-svgs/#aa-2-inline-svg), [W3 Reference](https://www.w3.org/TR/SVG11/struct.html#DescriptionAndTitleElements)

Reply:
I want to convert the svg to an image asset like the bevy logo anyway so that will work